.travis.yml

```diff
@@ -19,6 +19,7 @@ script:
 env:
   global:
     - CODACY_PROJECT_TOKEN=24f7bf62b3fa4328aa3b9bf32c07b0d9
+    - AZURE_APPLICATIONINSIGHTS_INSTRUMENTATIONKEY=SomeRandomString
 
```

Comment:
This can be set at docker-compose level as well. Not sure preferred one.

Reply:
Was that all that was needed?  I think better to sort out at at app level, rather have it running correctly and not expose app level things in travis.